scout/server/blueprints/variant/controllers.py

```diff
@@ -21,22 +21,25 @@
 LOG = logging.getLogger(__name__)
 
 def variant(store, institute_id, case_name, variant_id=None, variant_obj=None, add_case=True,
-            add_other=True, get_overlapping=True, add_compounds=True, variant_type=None):
+            add_other=True, get_overlapping=True, add_compounds=True, variant_type=None,
+            case_obj=None, institute_obj=None):
```

Comment:
Hm, I guess, but its a little redundant, isnt it. If it needs to stay this way for now, note in the docstring that the objects have precedence over the ids, even though the ids are mandatory and the objects are not.